### DISCUSSION TITLE
Asterisk pattern

### DISCUSSION THREAD

**Discussion by StoyanDimitrov** [dnGrep/dnGrep#558] [Q&A] [answered]
Isn't it what is referred to as "[asterisk pattern](/dnGrep/dnGrep/blob/bcf0c3862412a84e99150d876a40e964fddd65db/dnGREP.Localization/Properties/Resources.resx#L201)" known as "wildcard [characters] pattern matching" or just "wildcards"?

Fun note: Can you guess what results I got from searching the net for "asterisk pattern" as this term was previously not know to me?

**Comment by doug24** [comment, marked as answer]
Yes, that's right:  wildcard pattern matching.  

My first search hits are on a framework for building communications apps...
